### DISCUSSION TITLE
ALL Data Sorce Disconnected.

### DISCUSSION THREAD

**Discussion by giledi92** [utmstack/UTMStack#809] [Q&A]
Hello.

I have issue today in UTMSTACK 10.5.5, all data source is disconnected, the connection is established with the server but on the UTMSTACK shows it's not connected.  

![utmstack issue](https://github.com/user-attachments/assets/35d380f3-dbfd-4cec-b154-316974355d6b)

**Comment by Kbayero** [comment]
Hello @giledi92, this looks like your utmstack agent is failing. You can try to find some error message in 
```
/opt/utmstack-linux-agent/logs/utmstack_agent.log
```
 Also you can look for error messages in log auth proxy container. For this, you can use the command 
```
docker service logs utmstack_log-auth-proxy
```

**Reply by giledi92** [reply]
Hello , Thanks for your replay,  this is the output for the logs.

![utmstack_log-auth-proxy](https://github.com/user-attachments/assets/b094666b-7840-45d9-87ec-ec1ff37b420d)
![utmstack_agent log](https://github.com/user-attachments/assets/910f3cdf-9dbb-4a42-9037-8677a0b7eca4)
